### ISSUE TITLE
assemble nomenclature guidelines

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/monarch-legacy#696]
it would be really nice to assemble the various guidelines for different mods into one place, and provide the documentation on our website to help people understand the labels of the genotypes.  we could eventually write how these are similar/different, but for now, assembling links is a good place to start.

here's the [list assembled from HGNC](http://www.genenames.org/about/faq#Whatis):
Mouse: http://www.informatics.jax.org/mgihome/nomen/
Rat: http://rgd.mcw.edu/nomen/nomen.shtml 
Chicken: http://www.agnc.msstate.edu/
Anolis lizard http://lizardbase.org/pages/agnc.html
Xenopus: http://www.xenbase.org/gene/static/geneNomenclature.jsp
Zebrafish: https://wiki.zfin.org/display/general/ZFIN+Zebrafish+Nomenclature+Guidelines
Drosophila: http://flybase.org/static_pages/docs/nomenclature/nomenclature3.html
C. elegans: http://wiki.wormbase.org/index.php/Nomenclature
Arabidopsis: http://www.arabidopsis.org/portals/nomenclature/guidelines.jsp
S. cerevisiae: http://www.yeastgenome.org/gene_guidelines.shtml
S. pombe: http://www.pombase.org/submit-data/gene-naming-guidelines 

HGNC is the official human gene naming body. but here's other human genetic nomenclature guidelines:
http://www.hgvs.org/mutnomen/
http://www.cydas.org/Resources/ISCN_Discussion.html
a recent paper comparing ISCN 2009 vs 2013: https://www.karger.com/Article/FullText/353118

also for worm:
http://www.wormbase.org/about/userguide/nomenclature

**Comment by nlwashington** [comment]
some kind of summary of these in the GENO paper would be very helpful, i think @mbrush

**Comment by jmcmurry** [comment]
@mbrush could you first find the guidelines for the following:
- [ ] Gray Short-Tailed Opossum
- [ ] Platypus
- [ ] Horse
- [ ] Rhesus Monkey
- [ ] Chimpansee Troglodytes
- [ ] Canis Domesticus

@nlwashington could you clarify whether you intended for the info to be presented as a static HTML page or in a context-aware way. That is to say, should we be storing these in a machine readable representation so that we can  offer a "what's this" kind of tooltip for users?
